internal/policy/container/labels.go

```diff
@@ -0,0 +1,9 @@
+package container
```

Comment:
This file seems extraneous. If you want this to be a library function that can be re-used by both "prohibited" and "required" label checks, might be better to move this to another package. Here, it feels like cruft.

Reply:
@bcrochet suggested this, so that it's easier to find, since I had originally kept it in the original labels check.

Reply:
It's being used in two files. It does seem extra, but I actually thought there should probably be more shared code that could end up in here. I say establish this file now, and we refactor these two checks to strip out more of the shared code.

Reply:
Understood. The final point I'll stand on here is that `getDataForValidate` is a common name used through all checks. It's not a required naming convention by any means, but it's just been the common practice. It can change at any time.

That said, this name is used as a method on one of the Check types. Here, it's in the global namespace for the package, which I think makes it easier to confuse things.

Can we name this something _else_?

It's a bit of a nit, but if too many small nits get through, things can get messy fast.

Reply:
Renamed this to `getContainerLabels`